src/table/settings/economy_settings.ini

```
@@ -12,6 +12,8 @@ static void TownFoundingChanged(int32_t new_value);
 static void ChangeTimekeepingUnits(int32_t new_value);
 static void ChangeMinutesPerYear(int32_t new_value);
 
+static constexpr std::initializer_list<const char*> _place_houses{"forbidden", "allowed", "allowed, fully constructed"};
```

```suggestion
static constexpr std::initializer_list<const char*> _place_houses{"forbidden", "allowed", "fully constructed"};
```

The reason I think it is okay to not mention `allowed` here is because it is a setting, but I also think adding a comma here can make for a very lengthy setting value. But I am not sure that is actually a problem. So .. yeah, do with this what-ever you like :D I am not sure of my own comment :P